.ci_support/linux_64_c_compiler_version10cuda_compilernvcccuda_compiler_version11.1cxx_compiler_version10fortran_compiler_version10mpimpichnumpy1.21python3.10.____cpython.yaml

```diff
@@ -25,7 +25,7 @@ fortran_compiler:
 fortran_compiler_version:
 - '10'
 hdf5:
-- 1.14.0
+- 1.12.2
```

Comment:
This is happening as the `hdf5` migrator was replaced (hence that file was dropped). Have attempted to fix this